packages/product-react/src/components/Image/Image.tsx

```diff
@@ -0,0 +1,82 @@
+import clsx from 'clsx';
+import React from 'react';
+
+import './Image.scss';
+
+export type SizeVariations = '16by9' | '1by1' | '3by4' | '4by3' | 'cinema';
+
+export interface IImageProps extends React.HTMLProps<HTMLImageElement> {
+  className?: string;
+  src: string;
+  aspect: SizeVariations;
+  bg?: boolean;
+  alt?: string;
+  height?: number | string;
+  width?: number | string;
+  srcSet?: string;
+  useMap?: string;
+  caption?: string;
+  decoding?: 'async' | 'auto' | 'sync';
+}
+
+export const Image: React.FC<IImageProps> = ({
+  className,
+  src,
+  aspect,
+  alt,
+  bg,
+  height,
+  width,
+  srcSet,
+  useMap,
+  decoding,
+  caption,
+  ...rest
+}) => {
+  if (bg) {
+    return (
+      <div
+        className={clsx(
+          'ray-bg',
+          {
+            [`ray-bg--${aspect}`]: aspect
+          },
+          className
+        )}
+        {...rest}
+        style={{
+          ...rest.style,
```

Comment:
Seems like you should  pull style out via destructing above. That was you can just do `...style` And you don’t set this property and then overwrite it.

Reply:
Ya good catch. I fixed it thank bud